niworkflows/interfaces/masks.py

```diff
@@ -189,11 +189,11 @@ def _post_run_hook(self, runtime):
         ''' generates a report showing slices from each axis '''
 
         self._anat_file = self.inputs.realigned_file
-        self._mask_file = self.aggregate_outputs().high_variance_mask
-        self._seg_files = [self.aggregate_outputs().high_variance_mask]
+        self._mask_file = self.aggregate_outputs().high_variance_masks
+        self._seg_files = [self.aggregate_outputs().high_variance_masks]
```

Comment:
But in case of multiple masks this reportlet does not work well anyway (it ignores all but the first mask). I can add an assertion for such case.